ext/mysqlnd/mysqlnd_structs.h

```diff
@@ -936,6 +937,7 @@ struct st_mysqlnd_connection_data
 	unsigned int	protocol_version;
 	unsigned int	port;
 	zend_ulong		server_capabilities;
+        void*           sasl_connection;
```

Comment:
Adding a structure member in the middle of a structure changes the ABI. This means that the change should go into x.y.0 version of PHP or has the potential to break installations which use mysqlnd plugins. struct st_mysqlnd_connection_data is a structure which plugins often use.